src/ape/types/__init__.py

```diff
@@ -14,6 +14,8 @@
 )
 from hexbytes import HexBytes
 
+from ape.managers.query import AccountQuery, BlockQuery, EventQuery, MethodQuery
```

Comment:
Definitely don't import these here, this module must be "pure" of any classes from api or managers